src/mpid/ch4/netmod/ucx/subconfigure.m4

```diff
@@ -46,6 +46,19 @@ AM_COND_IF([BUILD_CH4_NETMOD_UCX],[
     else
         dnl PAC_PROBE_HEADER_LIB must've been successful
         AC_MSG_NOTICE([CH4 UCX Netmod:  Using an external ucx])
+
+        AC_MSG_CHECKING([if UCX meets minimum version requirement])
```

Comment:
moved version check to this block, which will throw an error if the external lib is not new enough, regardless of whether the user specified it or not.